### PR TITLE
Update wiki e212 scraper

### PR DESCRIPTION
**Pull request by cknv**
While it mostly works, there is a bunch of edgecases in the output
that we need to improve handling of, especially with overrides of
MCCs that are shared by multiple countries. It would be nice if the
wiki page was structured a bit differently, as that could make it
possible to extract the country codes from the page directly, but
that seems like a non-trivial task currently.

### CODE REVIEW

File: phone_iso3166/e212.py [hunk 1]
```diff
@@ -144,38 +425,35 @@
...
+        27: "IL",
+        28: "IL",
+        29: "IL",
     },
     426: "BH",
     427: "QA",
     428: "MN",
     429: "NP",
-    430: "AE",
-    431: "AE",
```
Comment (cknv) [inline phone_iso3166/e212.py]: These do exist, but as far as I can tell there is no networks to this MCC yet, so the wiki pages does not have any networks, and therefore they are skipped.

Should we have a list of additional MCC to countries we'd like to bolt on to this dataset @jwldk ?

File: phone_iso3166/e212.py [hunk 2]
```diff
@@ -84,6 +57,7 @@
     284: "BG",
     286: "TR",
     288: "FO",
+    289: "GE-AB",
```
Comment (jwldk) [inline phone_iso3166/e212.py]: Since the library is providing 2-letter country code from ISO 3166-1 we cannot support.

Comment (cknv) [reply phone_iso3166/e212.py]: Yeah, makes sense

**Review by jwldk — APPROVED** [review]
I think this is fine for a test release version.

**Comment by cknv** [comment]
I think the updated scraper and the override system is pretty decent, but given the imperfect mapping between countries and MCCs it is very likely that there is some errors, and I think those are very hard to detect.

All in all, I think this is probably about as good as what we already have, but at least more recent, so we can probably merge this and #27 and make a development release.

**Pull request merged**